doc/man1/flux-jobs.adoc

```diff
@@ -81,6 +81,16 @@ nnodes:: job node count (if job ran / is running), empty string otherwise
 nnodes_hyphen:: same as nnodes, but '-' if job has not run yet / never ran
 ranks:: job ranks (if job ran / is running), empty string otherwise
 ranks_hyphen:: same as ranks, but '-' if job has not run yet / never ran
+success:: True of False if job completed successfully, empty string otherwise
+success_hyphen:: same as success, but '-' if job has not yet completed
+exception:: True of False if job had an exception, empty string otherwise
+exception_hyphen:: same as exception, but '-' instead of empty string
+exception_severity:: If exception True, the highest severity, empty string otherwise
+exception_severity_hype:: same as exception_severity, but '-' instead of empty string
```

Comment:
LET'S GET HYPE!!!!!

Totally subjective but I think just the straight truncation (`exception_severity_hyph`) might be better here.

Reply:
Another random idea: rather than having a plethora of `_hyphen` attributes, I wonder if we could express this in the `format_spec` part of the format string.  For example `{exception_severity:h}` where `h` invokes a [custom conversion type](https://stackoverflow.com/questions/19864302/add-custom-conversion-types-for-string-formatting).

Reply:
> Another random idea: rather than having a plethora of _hyphen attributes, I wonder if we could express this in the format_spec part of the format string. For example {exception_severity:h} where h invokes a custom conversion type.

This is a great idea.  Not for this PR, wrote up issue #2859 for it